examples/net_client/resources/input.ron

```diff
@@ -0,0 +1,6 @@
+(
```

Comment:
You're not using input in your example so this file is unused.  Can be safely deleted,

Reply:
fixed